test/common/3Scale_user_promotion.go

```diff
@@ -11,10 +14,26 @@ import (
 )
 
 func Test3ScaleUserPromotion(t TestingTB, ctx *TestingContext) {
+	// To run this testcase for multiple test-users, set USER_NUMBERS to a string
+	// in a "1,2,3" format
+	userNumbers := os.Getenv("USER_NUMBERS")
+	testUserNumbers := strings.Split(userNumbers, ",")
+
+	if userNumbers == "" {
+		t.Logf("env var USER_NUMBERS was not set, defaulting to 1")
+		testUserNumbers = []string{"1"}
+	}
 	var (
-		developerUser      = fmt.Sprintf("%v%02d", DefaultTestUserName, 1)
-		dedicatedAdminUser = fmt.Sprintf("%v%02d", defaultDedicatedAdminName, 1)
+		developerUsers     []string
+		dedicatedAdminUser = fmt.Sprintf("%v%02d", defaultDedicatedAdminName, 2)
 	)
+	for _, numberString := range testUserNumbers {
+		number, err := strconv.Atoi(numberString)
+		if err != nil {
+			t.Fatalf("wrong format used for the `USER_NUMBERS` env variable %v", err)
```

Comment:
I think it would be good to say what the correct format is here (coma-separated list of numbers)